misc/acrn-config/schema/VMtypes.xsd

```diff
@@ -158,21 +181,21 @@
 
 <xs:complexType name="MmioResourcesConfiguration">
   <xs:sequence>
-<xs:element name="TPM2" type="Boolean" minOccurs="0" />
-<xs:element name="p2sb" type="Boolean" minOccurs="0" />
+<xs:element name="TPM2" type="Boolean" default="n" minOccurs="0" />
+<xs:element name="p2sb" type="Boolean" default="n" minOccurs="0" />
   </xs:sequence>
 </xs:complexType>
 
 <xs:complexType name="PciDevsConfiguration">
   <xs:sequence>
-<xs:element name="pci_dev" type="xs:string" maxOccurs="unbounded" />
+<xs:element name="pci_dev" type="xs:string" default="pci dev" maxOccurs="unbounded" />
   </xs:sequence>
 </xs:complexType>
 
 <xs:complexType name="BoardPrivateConfiguration">
   <xs:sequence>
-<xs:element name="rootfs" type="xs:string" />
-<xs:element name="bootargs" type="xs:string" />
+<xs:element name="rootfs" type="xs:string" default="/dev/nvme0n1p3"/>
+<xs:element name="bootargs" type="xs:string"/>
```

Comment:
I'm a bit confused about the coding style. Do we plan to add a space between the last tag and the end of a node, i.e. which is the preferred style, `... default="/dev/nvme0n1p3"/>` or `... default="/dev/nvme0n1p3" />`? We'd better keep consistent throughout the file.

Reply:
Tried to format across the files, make  <......../> to <........ />, means the backslash with space, but if there is no backslash, keep them no space. Tried to make all consistent, hope there is no one left out.

Reply:
Fine with me. Thanks.